### PR TITLE
add(locale): pt-br translate

### PR DESCRIPTION
**Pull request by mur4i**
## Description

add(locale): pt-br translate

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Comment by Manason** [comment]
Conflicts

File: server/commands.lua
Comment (Manason) [inline server/commands.lua:1]: Create new locales for this

**Comment by BerkieBb** [comment]
Still has conflicts and no response in 5 days for the review, in 1 week this will be closed if no action has been taken.

**Comment by Manason** [comment]
Closing. Feel free to re-open once feedback is addressed.

**Pull request closed without merging**